frame-metadata/src/lib.rs

```diff
@@ -441,13 +546,19 @@ impl Into<RuntimeMetadataPrefixed> for RuntimeMetadataLastVersion {
 
 /// All metadata about an runtime module.
 #[derive(Clone, PartialEq, Eq, Encode)]
-#[cfg_attr(feature = "std", derive(Decode, Serialize, Debug))]
+#[cfg_attr(feature = "std", derive(Decode, Serialize, Deserialize, Debug))]
 pub struct ModuleMetadata {
+	/// Name of the pallet.
 	pub name: DecodeDifferentStr,
+	/// Optional storage items.
 	pub storage: Option<DecodeDifferent<FnEncode<StorageMetadata>, StorageMetadata>>,
+	/// Public calls (dispatchables) to the pallet.
```

Comment:
```suggestion
	/// [`Option`] of public calls (dispatchables) to the pallet. See [`ODFnA`].
```
…or something to help people understand what `ODFnA` is.